### DISCUSSION TITLE
MosaicJSON POST endpoint?

### DISCUSSION THREAD

**Discussion by aliasmrchips** [developmentseed/titiler#281] [Q&A] [answered]
The MosaicJSON POST endpoint does not seem to be enabled. So, the MosaicJSON [example](https://developmentseed.org/titiler/examples/notebooks/Working_with_MosaicJSON/) doesn't work with an off-the-shelf deployment. Is there a way to enable this functionality and/or include MosaicJSON spec files in the deployed service?

**Comment by vincentsarago** [comment, marked as answer]
@aliasmrchips We removed it by default for security reason (without any customization e.g Token, it would enable anyone to create files on your S3 bucket). you can create a custom application and enable Mosaic creation using https://developmentseed.org/titiler/advanced/customization/#add-a-mosaicjson-creation-endpoint
